src/node.cc

```diff
@@ -2844,6 +2845,11 @@ void SetupProcessObject(Environment* env,
     // after LoadEnvironment() has run.
   }
 
+  // --track-heap-objects
+  if (track_heap_objects) {
+    READONLY_PROPERTY(process, "trackHeapObjects", True(env->isolate()));
+  }
```

Comment:
Yup. You're right. I can't think of a useful case to expose either of them to JS.